### ISSUE TITLE
"include" and "LRTalpha" argument

### ISSUE THREAD

**Issue by sdevlin3001** [cvoeten/buildmer#13]
Hello,

I would like to use buildmer as a means to identify significant level-2 predictors in my multilevel model. I already have a baseline multilevel model established, so I would like to retain all of the terms in this model and increase the significance level. Below is the line of code I am trying to run:

buildGLMMadaptive(Percent ~ (ot1 + ot2 + ot3 + ot4 + ot5 + ot1|ParticipantNumber), data=dat_all, family = "binomial", 
          buildmerControl = buildmerControl(include="ot2",direction="backward",list(args=list(nAGQ=0, LRTalpha(0.15)))))

This leads to a host of errors, but most notably it says "In buildmer.prep(match.call(), add = list(fit = fit.GLMMadaptive,: Passing extra arguments in '...' is deprecated, please use buildmerControl=list(args=list(...)) instead."  It actually runs when I remove the LRTalpha entirely, yet it does not retain any inputs in the "include"

So in summary (1) where do I include "LRTalpha" and (2) what is the proper format for "include"

Thank you!

**Comment by sdevlin3001** [comment]
So I got the "include" parameter to run as expected (the syntax is just args=list not an extra list() before that) but I am not sure if LRTapha() is being read when the elimination criteria is LRT? I say this because I get the same model results with LRTalpha is 0.05, 0.2, and 0.9 across multiple models. Any guidance would be much appreciated!

**Comment by cvoeten** [comment]
There are a couple of issues here.

1) 'include' should be specified as a one-sided formula: include=~a+b+c
2) LRTalpha() is an elimination criterion, i.e. use it as buildmerControl=list(elim=LRTalpha(.15))
3) Wrapping the extra arguments in an additional list will indeed not work as you intend. The nameless outer list will be matched to the 'formula' formal argument, and will subsequently be overwritten by the actual formula you specified. You've already figured out the right way to do it.

I don't think any of this is a bug in buildmer, apart from the fact that - evidently - the documentation is not clear enough about the syntax for these three situations. Could you give any suggestions on how I could improve the documentation?

**Comment by cvoeten** [comment]
Closing old.
